code/script.js

```diff
@@ -1,51 +1,270 @@
+}
+
+const userReply = (msg) => {
+  showMessage(msg, 'user')
+}
+
+// Removes bot typing effect
+
+const removeTyping = () => {
+  document.getElementById('preloader').style.display='none'
+}
 // This function will add a chat bubble in the correct place based on who the sender is
-const showMessage = (message, sender) => {
+//without the ${message} the actual message(parameter) in const greetUser will not be passed, only string message. Only the parameter 'bot' 
+function showMessage(message, sender) {
   // the if statement checks if the sender is 'user' and if that's the case it inserts an html senction inside the chat with the posted message
   if (sender === 'user') {
+    console.log("Now the user is replying");
     chat.innerHTML += `
       <section class="user-msg">
         <div class="bubble user-bubble">
           <p>${message}</p>
         </div>
-        <img src="assets/user.png" alt="User" />  
+        <img src="assets/ananasanswer.png" alt="User" />  
       </section>
-    `
-    // the else if statement checks if the sender is a bot and if that's the case it inserts an html senction inside the chat with the posted message
+    `;
+    // the else if statement checks if the sender is a bot and if that's the case it inserts an html section inside the chat with the posted message
   } else if (sender === 'bot') {
+    console.log("Now the bot is replying");
     chat.innerHTML += `
       <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
+        <img src="assets/ananasbot.png" alt="Bot" />
         <div class="bubble bot-bubble">
           <p>${message}</p>
         </div>
       </section>
-    `
+    `;
   }
   // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
   chat.scrollTop = chat.scrollHeight;
 }
 
-// Starts here
+//When website is loaded (we save), chatbot greets us:
+//We put in the parameter time and give it contidions. This is our input that controls which output we get.
+//We also put in the parameter timeGreetings, which is an array of greeting phrases. This becomes our output, depending on the time variable.
 const greetUser = () => {
-  // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot');
-  // Just to check it out, change 'bot' to 'user' here 👆
-}
 
-// Set up your eventlisteners here
+  botReply("Aloha what´s your Name?");
+
+};
 
-// When website loaded, chatbot asks first question.
-// normally we would invoke a function like this:
-// greeting()
-// But if we want to add a little delay to it, we can wrap it in a setTimeout:
-// setTimeout(functionName, timeToWaitInMilliSeconds)
-// This means the greeting function will be called one second after the website is loaded.
+//We welcome the user (based on current time) with a delay of 1s
```

Comment:
You do not greet the user based on curent time here right, or am I missing something? The delay is 1 sek but I cant see a specific time-greeting. If so I wanna know how (cause I want to learn! 😬)